### ISSUE TITLE
Check DPI scaling on windows

### ISSUE THREAD

**Issue by jlnr** [gosu/gosu#450]
Games shouldn't be comically small on a screen with 200% DPI scale on Windows. Not sure what to do at 125% and 150%? Just scale all the things, like in fullscreen mode?

**Comment by jlnr** [comment]
(Related: #438)

**Comment by Juksefantomet** [comment]
Are you really sure that it would really affect the examples at all unless the examples themselves are set to a higher resolution? 

I'm working on a 4k screen at work (ubuntu 18.04), regardless of dpi changes, if the resolution is set to less then 1200x1080 it still feels small, that is equivalent to 1/3 of the screen no matter how much you up the %. 

At home, ultra widescreen 2560x1080 (windows 10), even at that resolution i have issues with lower resolution stuff just looking plain weird. And if i blip it onto my tv with 4k, i need binoculars 😃 

The examples even at 200% would only be 640x480 * 2. meaning you will barely hit the 1/3 marker. 

i'd say one of the issues is the fact that the resolution itself is lower then the standards of todays resolution.

Fullscreen changes the resolution fixing that issue, but going windowed will not work unless the game has a base-resolution that is standard adaptable to at minimum 50% of the 4k resolution. (if that's the mark)

**Comment by jlnr** [comment]
True, the example games are pretty small by default. Nobody even remembers 640x480 as an actual screen resolution. On the other hand, users don't need to be immersed in the example games, and they might want to open them side-by-side with a text editor on a 13" laptop. Maybe I'll increase the resolutions when I work on gosu-examples again, but not by much.

**Comment by jlnr** [comment]
I've just tested this script on a mixed-DPI setup with Windows 10: https://github.com/gosu/gosu/pull/459#issuecomment-419735425

The window scales correctly (it gets larger), but it does not use the additional pixels – the 1px checkerboard is still grey at 200% scale. I think Gosu needs to [call this function](https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-setthreaddpiawarenesscontext) if there's no way to let SDL handle this for us. See also this [Win32 example project](https://github.com/Microsoft/Windows-classic-samples/tree/master/Samples/DPIAwarenessPerWindow).

**Comment by jlnr** [comment]
Also, `screen_{width,height}` returns *scaled* values right now on Windows, not pixel values. But maybe that will fix itself once I use the DPI awareness API.
